cmd/clusterctl/testdata/create-no-args-invalid-flag.golden

```diff
@@ -1,9 +1,6 @@
+Command "create" is deprecated, clusterctl has been deprecated in v1alpha2 and will be removed in a future version.
 Error: unknown flag: --invalid-flag
 Usage:
-  clusterctl create [command]
```

Comment:
I don't necessarily mind this diff, but this could cause issues for users of `clusterctl` in that the usage does not actually show the deprecated subcommands now.

Reply:
I had the same feeling about degrading the user experience of `clusterctl --help` commands. 

The behavior of `kubectl run` only shows the deprecation message when executing the command and not in any help or usage output. We can follow the same behavior, however I feel it would be useful to display the deprecation message as part of the help output. I have another solution where we use a custom help template. I'll re-submit the PR and we can see how that feels.